.golangci.yml

```diff
@@ -31,8 +33,8 @@ linters-settings:
       # - G304
       # - G404
   govet:
-    # report about shadowed variables
-    check-shadowing: true
+    enable:
+      - shadow
```

Comment:
This was generating a warning for being a deprecated means of enabling this rule.